backend/apps/investors/models.py

```diff
@@ -2,6 +2,8 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from phonenumber_field.modelfields import PhoneNumberField
+from apps.projects.models import Project
```

Comment:
This import introduces strong coupling and can produce circular import issues during Django startup and migrations. Models should generally reference other models by app label string to avoid import-time problems.

Replace direct import with lazy string reference in ForeignKey:
```
project = models.ForeignKey('projects.Project', on_delete=models.PROTECT, related_name='investments')
```
Remove the direct import to avoid circular import and keep module boundaries clean.